lib/remote_ip/debugger.ex

```diff
@@ -111,7 +111,7 @@ defmodule RemoteIp.Debugger do
   if Version.match?(System.version(), "~> 1.10") do
     @debug Application.compile_env(:remote_ip, :debug, false)
   else
-    @debug Application.get_env(:remote_ip, :debug, false)
+    @debug apply(Application, :get_env, [:remote_ip, :debug, false])
```

Comment:
I take it this is to circumvent the warning:
```
warning: Application.get_env/3 is discouraged in the module body, use Application.compile_env/3 instead
```

But actually, if you look at the enclosing `if`, you'll find that this line is only executed in versions of Elixir predating `Application.compile_env/3` (i.e., versions lower than 1.10). The last time I updated this, the minimum supported Elixir was 1.7, which meant we weren't guaranteed access to this function. See 7acd4e7289a634fffed5f55ab44c5464e8ced163.

However, now that the minimum supported version of Elixir is 1.10, we can remove the conditional altogether in favor of `Application.compile_env/3`. (This requires us to bump the Elixir version in the mix.exs file, as I mentioned earlier in this review.)